templates/python/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl

```diff
@@ -183,7 +186,7 @@
         ],
         "preLaunchTask": "Deploy (Test Tool)",
         "presentation": {
-            "group": "2-local",
+            "group": "0-TestTool",
```

Comment:
Is test tool still the second option since `order` is not changed?

Reply:
It will be the top priority order